framework/src/controller/controller.ts

```diff
@@ -119,7 +119,7 @@ export class Controller {
 
 		for (const alias of Object.keys(plugins)) {
 			const klass = plugins[alias];
-			const options = pluginOptions[alias];
+			const options = { dataPath: this.config.rootPath, ...pluginOptions[alias] };
```

Comment:
I think `this.config.rootPath` is not equivalent to `dataPath`.
if we add plugin options, maybe it's better to just pass all the root config and plugin config like below?
```
nodeConfig: {},
plugin: {},
```